fv3core/pace/fv3core/stencils/updatedzd.py

```diff
@@ -207,6 +208,10 @@ def __init__(
                 can be used as an approximation
             column_namelist: dictionary of parameter columns
         """
+        orchestrate(
+            obj=self,
+            config=stencil_factory.config.dace_config,
+        )
```

Comment:
It seems like every time `orchestrate` is called it uses keyword-only arguments, yet these are implemented as positional arguments in the definition of the function. Might I suggest switching the definition to,

```python
def orchestrate(*, obj, ...):
    ...
```

in `pace.dsl.dace.orchestration`.

Reply:
It really is just a copy-paste. I'd rather spell out the arguments in the definition. I dislike not having arguments laid out

Reply:
The arguments would still be laid out in the definition, the `*` in Python just forces the callers of the function to use keyword arguments for all remaining arguments.

Reply:
So, all your existing calls would remain correct (and would be the _only_ correct way to call it)

Reply:
That might be a good idea then, since this is quite the weird function overall